nestkernel/simulation_manager.h

```diff
@@ -210,6 +213,9 @@ class SimulationManager : public ManagerInterface
   Stopwatch sw_update_;
   Stopwatch sw_gather_target_data_;
 #endif
+
+  Time simulate_from_ = Time(Time::ms(0.0));
+  Time simulate_to_ = Time(Time::ms(0.0));
```

Comment:
Please move the initalization to the initializer list in the constructor together with intialization of all other fields. Also, add documentation for the fields.

Add spaces inside () to make the syntax checker happy.